### ISSUE TITLE
ObjectUnderTest should be preceded and followed by a blank line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#365]
A call on `ObjectUnderTest` (SUT) - may it be a property or a field - should be preceded by a blank line if the preceding line contains a call to something that is no call on `ObjectUnderTest`. In addition, a call to `ObjectUnderTest` should be followed by a blank line.

The reason is ease of reading (spotting test subject calls with ease).

Following should report a violation:
```c#
var x = 42;
var y = "something";
ObjectUnderTest.DoSomething(x, y);
```

While following should **NOT** report a violation:
```c#
var x = 42;
var y = "something";

ObjectUnderTest.DoSomething(x, y);
```

Following should report a violation:
```c#
ObjectUnderTest.DoSomething(x, y);
var x = 42;
var y = "something";
```

While following should **NOT** report a violation:
```c#
ObjectUnderTest.DoSomething(x, y);

var x = 42;
var y = "something";
```
